controllers/user.lua

```diff
@@ -235,19 +235,24 @@ UserController = {
                 )
             end
             assert_user_exists(self)
-            if self.queried_user.verified then
-                return user_page(self.queried_user)
-            end
 
             -- admins can verify people without the need of a token
             if self.params.token == '0' then assert_admin(self)
+                if self.queried_user.verified then
+                    return okResponseWithStatusCode('User ' .. self.queried_user.username ..
```

Comment:
Hmm, what's the reason for the new method, and the 'Already Verified' argument? The `status_code` argument of the `jsonResponse` method is meant to be a HTTP Status code, in this case it should always be 200. 

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status

Reply:
It was meant to be a message representing if the user was verified successfully or already verified, so the bulk verify can let the verifying person know who got verified or who was already verified
There isn't really a way of distinguishing between either one unless one were to use the message string, which is prone to be modified in the future since it is a message

I changed the name to make it less confusing between a Request status code and the Status of reaching this point of the code (I called it the State Message)